Sources/Danger/Plugins/SwiftLint/ShellExecutor.swift

```diff
@@ -31,16 +22,8 @@ internal class ShellExecutor {
 
     // Similar to above, but can throw, and throws with most of
     // what you'd probably need in a scripting environment
-    func spawn(_ command: String, arguments: [String] = [], environmentVariables: [String] = []) throws -> String {
-        let script = [environmentVariables.joined(separator: " "),
-                      command,
-                      arguments.joined(separator: " ")].filter { !$0.isEmpty }.joined(separator: " ")
-
-        var env = ProcessInfo.processInfo.environment
-        let task = Process()
-        task.launchPath = env["SHELL"]
-        task.arguments = ["-l", "-c", script]
-        task.currentDirectoryPath = FileManager.default.currentDirectoryPath
+    func spawn(_ command: String, arguments: [String] = [], environmentVariables: [String: String] = [:]) throws -> String {
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-default;
  File: Sources/Danger/Plugins/SwiftLint/ShellExecutor.swift;
  Line: 25;
-->

- :warning: Line should be 120 characters or less: currently 124 characters (`line_length`)